inline-images.rb

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2019 The Khronos Group Inc.
+
+require 'asciidoctor/extensions' unless RUBY_ENGINE == 'opal'
+require 'base64'
+
+Asciidoctor::Extensions.register do
+  postprocessor do
+    process do |document, output|
+      _replace_urls(output) if document.basebackend? 'html'
+    end
+  end
+end
+
+MIME_TYPES = {
+    '.png' => 'image/png',
+    '.svg' => 'image/svg+xml',
+    '.ico' => 'image/x-icon'}.freeze
```

Comment:
Nit. Please move closing `}.freeze` to a new line. It is too easy to miss the closing now.